package.json

```diff
@@ -2,20 +2,40 @@
   "name": "@cheqd/mcp",
   "version": "1.0.0",
   "description": "Package with all the mcp toolkits for cheqd",
-  "workspaces": [
```

Comment:
Isn't this necessary?

Reply:
> Isn't this necessary?

No it's not necessary for pnpm. We have a separate pnpm-config.yaml